ci/scripts/common.sh

```
@@ -106,20 +106,26 @@ function get_num_proc() {
-   set -e
-
-   if [[ ${SETUPTOOLS_SCM_RESULT} -ne 0 ]]; then
-       rapids-logger "Error, setuptools_scm failed to determine the version: ${SETUPTOOLS_SCM_OUTPUT}"
-       exit ${SETUPTOOLS_SCM_RESULT}
+
+   if [[ "${CI_CRON_NIGHTLY}" == "1" || "${IS_TAGGED}" == "1" ]]; then
+      # For tagged releases and nightly builds, use the git tag as the version as-is
+      local NAT_VERSION="${GIT_TAG}"
+   else
+      set +e
+      local NAT_VERSION=$(python -m setuptools_scm)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)